autoPyTorch/utils/backend.py

```diff
@@ -205,12 +205,12 @@ def temporary_directory(self) -> str:
 
     def _make_internals_directory(self) -> None:
         try:
-            os.makedirs(self.internals_directory)
+            os.makedirs(self.internals_directory, exist_ok=True)
```

Comment:
probably we should warn or ask users if we should allow to overwrite log files

Reply:
Yeah I had a warning in the PR I made to automl common. Maybe we can copy it as is.

Reply:
Would you like to commit it then @ravinkohli ?

Reply:
actually, I didn't do it there. but I can add it definitely.

Reply:
I have done that.

Reply:
Thanks.